src/EnergyPlus/FluidCoolers.cc

```diff
@@ -635,6 +531,160 @@ namespace FluidCoolers {
 
 	}
 
+	bool
+	TestFluidCoolerTwoSpeedInputForDesign( 
+	std::string const cCurrentModuleObject,
+	FArray1D<std::string> const &  AlphArray,
+	FArray1D<std::string> const & cNumericFieldNames, 
+	FArray1D<std::string> const & cAlphaFieldNames,
+	int const &	FluidCoolerNum 
+	)
+	{
+		// FUNCTION INFORMATION:
+		//       AUTHOR:          Chandan Sharma
+		//       DATE WRITTEN:    August 2008
+		//       MODIFIED         Chandan Sharma, FSEC, April 2010
+		//       RE-ENGINEERED    Jason Glazer, GARD Analytics, February 2015, refactor into a separate function
+
+		// PURPOSE OF THIS FUNCTION:
+		// Separate the testing of inputs related to design so that it could be called from the unit tests
```

Comment:
@JasonGlazer Same thing here as the other pull request. Are you OK with this info?